JobConfig/beam/beam_defs_g4s1.fcl

```diff
@@ -98,7 +98,7 @@ source : {
 services : {
 
    message               : @local::default_message
-   TFileService          : { }
+   TFileService          : { fileName : "beamg4s1.root" }
    RandomNumberGenerator : {defaultEngineKind: "MixMaxRng" }
```

Comment:
I suspect that this field and the output file fields were left blank on purpose because generate_fcl fills them in.  The idea is that this is a deliberately incomplete file.   We do not want to force people to think about certain things when they use this fcl file - so we deliberately leave them blank.

I will add Andrei and Ray as a reviewers since they set up a lot of JobConfig.

Please check the whole pull request for this.

Reply:
This was the only file that I made these types of changes in.

Reply:
> I suspect that this field and the output file fields were left blank on purpose because generate_fcl fills them in. The idea is that this is a deliberately incomplete file. We do not want to force people to think about certain things when they use this fcl file - so we deliberately leave them blank.

Yes, files with "defs" in their names are supposed to be incomplete.  I think they even predate generate_fcl.  The "defs" are similar in spirit to what we call "prolog.fcl" now, but they were written before FHICL had all the functionality we can use now.